testapp/testapp/settings.py

```diff
@@ -0,0 +1,162 @@
+"""
+Django settings for testapp project.
+
+Generated by 'django-admin startproject' using Django 3.2.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import os
+
+PROJECT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+BASE_DIR = os.path.dirname(PROJECT_DIR)
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-y3y-&))dvmo-q5_zs88oljoq()y%7!g4870p^4!oz-22rq0t=!"
+
+# SECURITY WARNING: define the correct hosts in production!
+ALLOWED_HOSTS = ["*"]
+
+EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    "liveblog",
```

Comment:
Is `liveblog` a good name? Maybe it is bit confusing? `liveblog` is pretty similar to names you would encounter in the Wagtail Live package.

Maybe we can just move all content from liveblog and put it directly in the `testapp` folder. Then `INSTALLED_APPS` should have `'testapp',`.

Reply:
Alright! Actually the problem I have is handling migrations and models. That's the main reason I haven't still added tests for `LivePageMixin`. 

How do we create models/tables for tests? I tried to use fixtures but I would prefer another approach.